### PR TITLE
dns/eve: add 'HTTPS' type logging

### PR DESCRIPTION
**Pull request by hsadia538**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [X] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [X] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [ ] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/issues/4751) ticket.

Previous PR: https://github.com/OISF/suricata/pull/8076

Describe new changes:

- Fix `output-json-dns.c `

suricata-verify-pr: 975

### CODE REVIEW

File: rust/src/dns/log.rs [hunk 1]
```diff
@@ -86,6 +86,7 @@ pub const LOG_URI        : u64 = BIT_U64!(59);
 
 pub const LOG_FORMAT_GROUPED  : u64 = BIT_U64!(60);
 pub const LOG_FORMAT_DETAILED : u64 = BIT_U64!(61);
+pub const LOG_HTTPS      : u64 = BIT_U64!(62);
```
Comment (jasonish) [inline rust/src/dns/log.rs:89]: Can we remove these LOG_HTTPS entries? I see it being set, but never read suggesting its not needed at this time.

Comment (jasonish) [reply rust/src/dns/log.rs:89]: Ah, maybe I'm wrong. IMO, an example of too much configuration for a protocol, and I've forgotten how it works.

Comment (hsadia538) [reply rust/src/dns/log.rs:89]: It is getting used on line 255

**Review by jasonish — APPROVED** [review]
Approved. CI failures are not the fault of this PR.

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #8097, thanks for your contribution!

**Pull request closed without merging**